src/git-source-provider.ts

```diff
@@ -0,0 +1,209 @@
+import * as core from '@actions/core'
+import * as coreCommand from '@actions/core/lib/command'
+import * as fs from 'fs'
+import * as fsHelper from './fs-helper'
+import {IGitCommandManager} from './git-command-manager'
+import * as gitCommandManager from './git-command-manager'
+import * as io from '@actions/io'
+import * as path from 'path'
+import * as refHelper from './ref-helper'
+
+const authConfigKey = `http.https://github.com/.extraheader`
+
+export async function getSource(
+  repositoryPath: string,
+  repositoryOwner: string,
+  repositoryName: string,
+  ref: string,
+  commit: string,
+  clean: boolean,
+  fetchDepth: number,
+  lfs: boolean,
+  accessToken: string
+): Promise<void> {
+  core.info(`Syncing repository: ${repositoryOwner}/${repositoryName}`)
```

Comment:
you're missing a lot of line ending semicolons. not sure if this is intentional or not

Reply:
that's the prettier config. personally i think its weird but i just copied the config from the toolkit 🤷‍♂